python/pyarrow/parquet.py

```diff
@@ -500,49 +502,28 @@ def __str__(self):
 
         return result
 
-    def get_metadata(self, open_file_func=None):
```

Comment:
This was still being used in dask 2.1.0, released a month ago: https://github.com/dask/dask/blob/ed33fbe6ec47e361d1f6f45b84acfe0a98e511ca/dask/dataframe/io/parquet.py#L860. But, it's fixed in the latest release 2.2 released a few days ago. So it might be fine to remove, but just that we are aware it was only fixed in dask *very* recently.

Reply:
Noted. This was deprecated in 0.13.0 so I think it's OK to remove since we had 2 major releases with the deprecated API and warning